docs/mindsdb-docs/docs/sql/create/predictor.md

```diff
@@ -2,10 +2,12 @@
 
 ## Description
 
-The `CREATE PREDICTOR` statement is used to train a new model. The basic syntax for training a model is as follows:
+The `CREATE PREDICTOR` statement creates and trains a new ML model.
 
 ## Syntax
 
+Here is the syntax:
+
 ```sql
 CREATE PREDICTOR mindsdb.[predictor_name]
 FROM [integration_name]
```

Comment:
Since here is full syntax, let add to the end `[ USING  parameters [, ...] ]`

Reply:
Added. Here is the full syntax now:
```
CREATE PREDICTOR mindsdb.[predictor_name]
FROM [integration_name]
    (SELECT [column_name, ...] FROM [table_name])
PREDICT [target_column]
[USING [parameter_key] = ['parameter_value']];
```